frontend/src/app/services/auth.service.ts

```diff
@@ -1,30 +1,33 @@
-import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
-import { BehaviorSubject } from 'rxjs';
-import { Router } from '@angular/router';
+import { Injectable } from "@angular/core";
+import { HttpClient } from "@angular/common/http";
+import { BehaviorSubject } from "rxjs";
+import { Router } from "@angular/router";
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: "root",
 })
 export class AuthService {
-
   isAuth$ = new BehaviorSubject<boolean>(false);
   private authToken: string;
   private userId: string;
 
-  constructor(private http: HttpClient,
-              private router: Router) {}
+  constructor(private http: HttpClient, private router: Router) {}
 
   createUser(email: string, password: string) {
     return new Promise((resolve, reject) => {
-      this.http.post('http://localhost:3000/api/auth/signup', {email: email, password: password}).subscribe(
-        (response: { message: string }) => {
-          resolve(response);
-        },
-        (error) => {
-          reject(error);
-        }
-      );
+      this.http
+        .post("http://localhost:3000/api/auth/signup", {
+          email: email,
+          password: password,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/AlchTech/JeremySalettes-Wozniak_6_02062021/pullRequest?prid=7545724)
```suggestion
          password,
```